pages/_error.tsx

```diff
@@ -0,0 +1,52 @@
+// CSS
+import styles from '../styles/Error.module.scss';
+// Components
+import NavBar from '../components/NavBar';
+import Button, { ButtonType } from '../components/Button';
+
+export default function ErrorPage({ 
+  statusCode 
+}:{
+  statusCode: any
+}) {
+  const returnTo = '/';
+
+  return (
+    <div className={styles.canvas}>
+      <NavBar currentPage='Home' />
+      <div className={styles.page}>
+        <div className={styles.container}>
+          <h1 className={styles.heading}>
```

Comment:
instead of using a class here maybe use css nesting something like:
```scss
.container {
  & > h1 {} // styling
}
```
The reason being we should enforce styles based off semantics and classes should add more context if they do not add any context why do we have the class hope that makes sense.